### DISCUSSION TITLE
Cinayeti kim işledi kripto

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#31] [Zeka Sorusu] [answered]
Polisler ihbar üzerine bir eve giderler. Evde bir ceset görürler ve intihar vakası diyerek olay yerinden ayrılırlar. Aynı günün ilerleyen vakitlerin de ölen kişinin bir arkadaşı intihar değil cinayet olduğundan şüphelenerek bir dedektif tutar. Dedektif Holmes olay yerini incelediğin de polislerin farkedemediği bir not bulur ve notta şunlar yazılı:
"6,3:_:2,2:3,2:6,2:4,3:6,1:_:5,2:2,1:7,3:4,3:6,1:3,1:4,3:!"
Dedektif Holmes polise bulduğu yazıyı bildirmek için cep telefonunu eline almıştı ki bir an da "BULDUM" diye bağırdı.
Cinayeti kim işledi ve dedektif bu olayı nasıl çözdü?

**Comment by OmerAkbayrak** [comment, marked as answer]
Dedektif şifreyi telefonu eline alıp polisi aramaya çalıştığında çözdüğü için arama klavyesindeki rakamlar ve üzerinde yazan harflerle ilgili olabileceğini düşündüm. Notta bulunan rakamları inceleyerek iki nokta üst üste işaretlerinin arasındaki 6,3  2,2   3,2 gibi rakamların klavyede hangi harflere denk geldiğini bulmaya çalıştım ve sonunda 6 numaralı tuşun 3. harfi = 6,3 ,   2 numaralı tuşun 2. harfi=2,2   şeklinde ilerleyince harfler sırasıyla "O benim karımdı." şifresi ortaya çıktı. Böylece katilin eski karısı olduğunu anlamış oldum.
